x-pack/plugins/cross_cluster_replication/public/__jest__/client_integration/follower_indices_list.test.js

```diff
@@ -379,16 +379,12 @@ describe('<FollowerIndicesList />', () => {
       test('should have a section to render the follower index shards stats', async () => {
         await actions.clickFollowerIndexAt(0);
         expect(exists('followerIndexDetail.shardsStatsSection')).toBe(true);
-      });
-
-      test('should render a EuiCodeEditor for each shards stats', async () => {
-        await actions.clickFollowerIndexAt(0);
 
-        const codeEditors = component.find(`EuiCodeEditor`);
+        const codeBlocks = component.find('EuiCodeBlock');
```

Comment:
Wouldn't it be better to access the code blocks with the test subject `shardsStats`?

```js
const codeBlocks = find('shardsStats');
```